### DISCUSSION TITLE
ffmpeg_avcodec out of order errors

### DISCUSSION THREAD

**Discussion by xraynorm** [Motion-Project/motion#1615] [Q&A]
Hello 

After investigating why I was missing video at the start of motion events, I noticed I am getting a lot of "out of order" errors in the motion log file. I have recently updated to Motion 4.5.1, so I am not sure if this is a hardware or software issue.

Are this errors important?

Reolink RLC-410 (RTMP stream) with pass through 

`[3:nc3] [INF] [ENC] [Jan 02 11:40:46] ffmpeg_avcodec_log: DTS 9575644 < 9576345 out of order`

Thanks

**Comment by Mr-Dave** [comment]
This looks to be a Motion software issue with the pass through for this camera.  

Unfortunately, there isn't a straight forward resolution for the situation when then DTS is different than the PTS.  (Setting of the PTS is complicated to start with on pass through and having a different DTS that doesn't correlate with real time is a really unique problem on top of that.)
